jupyterlab-dataops/Dockerfile

```diff
@@ -16,12 +16,44 @@ COPY requirements.txt requirements-${PROVIDER}.tx[t] $HOME/
 
 RUN apt-get update --fix-missing && apt-get -qq -y install \
     curl \
+    gnupg2
+
+RUN sh -c 'curl -sL https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add - && \
+    echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list && \
+    curl -sL https://deb.nodesource.com/setup_16.x | sudo -E bash -'
+
+RUN apt-get update --fix-missing && apt-get -qq -y install \
     unzip \
     zip \
     jq \
-    git && \
+    git \
+    openjdk-11-jdk \
+    maven \
+    yarn \
+    nodejs && \
     rm /bin/sh && ln -s /bin/bash /bin/sh
 
+# Cloudbeaver needs newer node
+RUN ln -sf /bin/node /opt/conda/bin/node
+RUN ln -sf /bin/npm /opt/conda/bin/npm
+
+RUN npm install -g lerna
+
+# Clone cloudbeaver
+RUN git clone https://github.com/dbeaver/cloudbeaver.git /tmp/cloudbeaver && \
+    cd /tmp/cloudbeaver
+
+# Add BigQuery drivers
+RUN mkdir -p /tmp/cloudbeaver/server/drivers/bigquery/src/main/resources
+ADD resources/cloudbeaver/plugin.xml /tmp/cloudbeaver/server/bundles/io.cloudbeaver.resources.drivers.base/plugin.xml
+ADD resources/cloudbeaver/drivers/pom.xml /tmp/cloudbeaver/server/drivers/pom.xml
+ADD resources/cloudbeaver/drivers/bigquery/pom.xml /tmp/cloudbeaver/server/drivers/bigquery/pom.xml
+RUN wget https://storage.googleapis.com/simba-bq-release/jdbc/SimbaJDBCDriverforGoogleBigQuery42_1.2.16.1020.zip && \
+    unzip SimbaJDBCDriverforGoogleBigQuery42_1.2.16.1020.zip -d /tmp/cloudbeaver/server/drivers/bigquery/src/main/resources && \
+    rm SimbaJDBCDriverforGoogleBigQuery42_1.2.16.1020.zip
```

Comment:
BigQuery / gcp can't be hardcoded this way. Look at github actions pipliens, platform is driven this way:     
strategy:
      matrix:
        provider: ["bigquery"]